### PR TITLE
feat: Products page

### PR DESCRIPTION
**Pull request by ozairkhan2810**
Added products page with filter

image

### CODE REVIEW

File: html/product.html [hunk 1]
```diff
@@ -0,0 +1,304 @@
...
+            </div>
+        </main>
+    </div>
+    <footer class="flex flex-column justify-center align-center gap-1">
+        <div class="white bold-500">
+            All rights reserved by <span class="red">CricKit</span> | Created by Ozair Khan
+        </div>
+        <div class="flex gap-1">
+            <div class="facebook">
+                <a href=""><img class="svg-icon" src="../assets/icons-svg/facebook-app.svg" alt="facebook" /></a>
```
Comment (Kothalkarpriya) [inline html/product.html]: use # to make dead link everytime

File: html/product.html [hunk 2]
```diff
@@ -0,0 +1,304 @@
...
+            </div>
+            <div class="instagram">
+                <a href=""><img class="svg-icon" src="../assets/icons-svg/ig-instagram.svg" alt="instagram" /></a>
+            </div>
+            <div class="twitter">
+                <a href=""><img class="svg-icon" src="../assets/icons-svg/twitter-app.svg" alt="twitter" /></a>
+            </div>
```
Comment (Kothalkarpriya) [inline html/product.html:296]: every time you used anchor tag do replace blank quotes with # please

**Review by fahd23 — APPROVED** [review]
Looks good, you can merge it

**Review by Kothalkarpriya — APPROVED** [review]
After making suggested changes you can merge

File: html/product.html (hunk 1 above)
Comment (ozairkhan2810) [reply html/product.html]: ok got it
Thanks for suggestion 👍

**Pull request merged**